enterprise/backup.md

```diff
@@ -8,15 +8,18 @@ keywords: enterprise, backup, restore
 To backup Docker Enterprise Edition you need to create individual backups
 for each of the following components:
 
-1. Docker Engine. [Learn more](/engine/swarm/admin_guide/#back-up-the-swarm).
-2. Universal Control Plane (UCP). [Learn more](/datacenter/ucp/2.2/guides/admin/backups-and-disaster-recovery/#backup-command).
-3. Docker Trusted Registry (DTR). [Learn more](/datacenter/dtr/2.3/guides/admin/backups-and-disaster-recovery/#backup-dtr-data).
+1. Docker Swarm. [Backup Swarm resources](/engine/swarm/admin_guide.md#back-up-the-swarm).
```

Comment:
Being more explicit, this is about backing up user app definitions running on Swarm, right? Paging @aluzzardi to confirm, and what else we might be missing